contracts/script/LightClient.s.sol

```diff
@@ -7,7 +7,7 @@ import { LightClient as LC } from "../src/LightClient.sol";
 import { ERC1967Proxy } from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
 
 contract DeployLightClientContractScript is Script {
-    function run(uint32 numBlocksPerEpoch, uint32 numInitValidators)
+    function run(uint32 numBlocksPerEpoch, uint32 numInitValidators, uint32 maxHistorySeconds)
```

Comment:
```suggestion
    function run(uint32 numBlocksPerEpoch, uint32 numInitValidators, uint32 stateHistoryRetentionPeriod)
```
Appears elsewhere too, need to do search/replace.

Reply:
thanks, sorted https://github.com/EspressoSystems/espresso-sequencer/pull/1670/commits/33663d70d64e7f79785d992b90d9dfc97280bc79